css/style.css

```diff
@@ -0,0 +1,116 @@
+
+body {
+  background-color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  padding: 0;
+  margin: 0;
+}
+
+/* HEADER-MOBILE */
+#main-container nav .mobile .header-mobile {
+  padding: 0;
+  margin: 1px;
+  display: flex;
+  width: 100;
+  height: 1fr;
+  background-color: rgb(255, 255, 255);
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+#main-container header .introduction h1 {
+  font-size: 52px;
+  text-align: center;
+  color: #172b4b;
+
+  /* padding-top: 70px ; */
+  margin: 70px 45px 10px 45px;
+}
+
+#main-container nav .mobile .header-mobile h1 {
+  font-size: 30px;
+  margin-left: 35px;
+  margin-top: 30px;
+  color: rgba(0, 0, 0, 0.774);
+}
+
+#main-container nav .mobile .header-mobile #bmenu {
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+}
+
+#main-container nav .mobile .header-mobile #bmenu .material-symbols-outlined {
+  margin: 10px 40px 0 0;
+  font-size: 35px;
+  color: black;
+}
+
+/* INTRODUCCTION-SECTION */
+#main-container .introduction {
+  background-image: url(../img/background-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
+#main-container header .introduction {
+  display: inline-block;
+}
+
+#main-container header .introduction h2 {
+  font-size: 52px;
+  text-align: center;
+  color: #36b37e;
+  margin: 10px 45px 70px 45px;
+}
+
+#main-container header .introduction p {
+  font-size: 25px;
+  text-align: center;
+  color: rgba(0 0 0 0.712);
+  margin: 0 30px 70px 30px;
+}
+
+#main-container header .introduction .social-button {
+  display: flex;
+  justify-content: center;
+  margin-top: 100px;
+  margin-bottom: 100px;
+}
+
+#main-container header .introduction .social-button .social-t {
+  cursor: pointer;
+  font-size: 40px;
+  border: none;
+  background-color: transparent;
+}
```

Comment:
- [ ] Please make your social icons smaller a little bit to match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170974590-efb2788f-4f65-46f7-9e80-73c4b845f83b.png)  | ![image](https://user-images.githubusercontent.com/68796232/170974566-3ded006e-ba5e-41ca-85cc-ed33799175e6.png) |